stdlib/LinearAlgebra/test/bidiag.jl

```diff
@@ -664,5 +664,30 @@ using .Main.ImmutableArrays
     @test convert(AbstractArray{Float64}, Bl)::Bidiagonal{Float64,ImmutableArray{Float64,1,Array{Float64,1}}} == Bl
     @test convert(AbstractMatrix{Float64}, Bl)::Bidiagonal{Float64,ImmutableArray{Float64,1,Array{Float64,1}}} == Bl
 end
+                    
+@testset "Tri/Bidiagonal conversion" begin
```

Comment:
I don't think this whole testset adds any new tests, all of these methods seem well covered already.